containers/ddev-dbserver/files/create_base_db.sh

```diff
@@ -71,24 +73,24 @@ mysql -uroot <<EOF
 	FLUSH TABLES;
 EOF
 
-mysqladmin -uroot password root
+mysqladmin -uroot --socket=${MYSQL_UNIX_PORT} password root
 
-if [  "${mysqld_version%%%.*}" = "8.0" ]; then
-    mysql -uroot -proot <<EOF
-        ALTER USER 'db'@'%' IDENTIFIED WITH mysql_native_password BY '$MYSQL_PASSWORD';
-        ALTER USER 'root'@'%' IDENTIFIED WITH mysql_native_password BY '$MYSQL_ROOT_PASSWORD';
-        ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY '$MYSQL_ROOT_PASSWORD';
+if [[  "${mysqld_version%%%.*}" = ^8.[04]$ ]]; then
```

Comment:
Should be this?
```suggestion
if [[  "${mysqld_version%%%.*}" =~ ^8.[04]$ ]]; then
```

Reply:
I wonder how we got by without this working.